README.md

```
@@ -1,3 +1,5 @@
+[![MseeP.ai Security Assessment Badge](https://mseep.net/pr/mawazawa-agentic-pdf-app-badge.png)](https://mseep.ai/app/mawazawa-agentic-pdf-app)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)